SwiftPackage/Sources/AssessmentModelUI/ViewModels/NodeState.swift

```diff
@@ -99,8 +99,9 @@ public final class AssessmentState : BranchState {
     @Published public var navigationError: Error?
 
     public init(_ assessment: Assessment, restoredResult: AssessmentResult? = nil, interuptionHandling: InterruptionHandling? = nil) {
-        let result = restoredResult ?? assessment.instantiateAssessmentResult()
-        self.interuptionHandling = interuptionHandling ?? assessment.interruptionHandling
+        let rules = interuptionHandling ?? assessment.interruptionHandling
```

Comment:
I'm not sure how to feel about using a misspelling to distinguish a parameter from the corresponding property but ok I guess--it triggers my proofreading reflex but admittedly that's a "me" thing

Reply:
Oops, I missed *that* misspelling. *sigh*